pkg/client/automation/client.go

```diff
@@ -267,3 +280,18 @@ func rmIDField(data *[]byte) error {
 	}
 	return nil
 }
+
+func get(a Client, resourceType ResourceType, offset int) (rest.Response, error) {
+
+	params := url.Values{}
+	params.Add("offset", strconv.Itoa(offset))
+
+	u, _ := url.ParseRequestURI(a.url)
```

Comment:
please just return that error if happens - even if it's pretty unlikely to occur, we should IMO not omit errors if we could as well handle them

Reply:
done